cli.json

```diff
@@ -5,9 +5,15 @@
   "commands": [
     {
       "name": "edgeworkers",
-      "version": "0.4.1",
+      "version": "0.5.0",
       "aliases": ["ew", "edgeworkers"],
       "description": "Manage Akamai EdgeWorkers code bundles."
+    },
+    {
+      "name": "edgekv",
+      "version": "0.5.0",
+      "aliases": ["ekv", "edgekv"],
+      "description": "Manage Akamai EdgeKV code bundles."
```

Comment:
Update description to be specific to EKV (it doesn't have code bundles :-) )

Reply:
Agreed. Updating to "Manage Akamai EdgeKV database"